### DISCUSSION TITLE
Wobble with AnyV2 models and AnyV2n

### DISCUSSION THREAD

**Discussion by loawizard** [nagadomi/nunif#132] [General]
When using the anyV2 and AnyV2N models you can notice that it creates a wobble in the image. It happens a lot.
I wonder what can be done to remove the wobbling effect?

**Comment by diverswan9** [comment]
I noticed that as well, though I would call it 'warping'.

**Comment by nagadomi** [comment]
I have not yet confirmed this issue.
Need to clarify if this is a problem with the pretrained-model or a bug in my implementation.
If you can see the same problem in the official video demo, it is a problem with the pretrained-model.

**Comment by nagadomi** [comment]
For Any_V2_N, there may be Depth Flicker problem. When negative foreground-scale it will be amplified.
Depth flicker is disparity(parallax) flicker on SBS video that causes the image to vibrate to the left or right.
https://github.com/nagadomi/nunif/discussions/110 Flicker Reduction may be able to reduce it.

**Reply by kaelsonofkrypto** [reply]
hey i know this is unrelated to this , but quick question, i noticed iw3 has an option that says all cuda devices, does this mean if i have two gpus installed on the same pc, it will use both gpus if i have the option selected?

**Reply by nagadomi** [reply]
Yes. batch-size is divided into GPUs and processed in parallel.
However, there are reports that it is not very effective on Windows WDDM driver. #59
Also, ZoeD_Any_N and ZoeD_Any_K are not supported. Other models are supported.

**Reply by kaelsonofkrypto** [reply]
Thanks
